.gitignore

```diff
@@ -103,8 +103,6 @@ venv.bak/
 # mypy
 .mypy_cache/
 
-#vscode
```

Comment:
I believe we can exclude only `launch.json` and `tasks.json` from `.gitignoge`. Personal setting should be ignored.